ftw/billboard/profiles/uninstall/portlets.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0"?>
+<portlets xmlns:i18n="http://namespaces.zope.org/i18n"
+          i18n:domain="plone">
+    <portlet
+      addview="billboard.actions.portlet"
+      title="billboard actions portlet"
+      description="display add/edid/del actions" remove="True">
+    </portlet>
+    
+    <assignment 
+        name="billboard-actions"
+        category="content_type"
+        key="BillboardCategory"
+        manager="plone.rightcolumn"
+        type="ftw.billboard.portlets.actions"
+        weight="15"
```

Comment:
what happens if someone changed the weight?

Reply:
The portlet gets removed... the value there is probably not necessary. and the uninstall test says the portlet is gone afterwards. Or I don't understand what you mean?

Reply:
I just wondered if `remove` only works if all properties mentioned match.

Reply:
ohh I don't know...

Reply:
@djowett-ftw I removed the weight attribute from the uninstall portlets.xml

Reply:
Great - thanks!